assembl/static2/js/app/components/debate/thread/post.jsx

```diff
@@ -167,6 +171,29 @@ class Post extends React.PureComponent {
     };
     return (
       <div className="posts" id={id}>
+        {extracts.length
+          ? <div className="extracts" style={{ position: 'absolute', left: '100px', width: '250px', lineHeight: '30px' }}>
```

Comment:
The styles here, don't look to be changing dynamically. You should instead either add these styles to the css class `extracts` or create a new class and add that to the list of classnames.

Reply:
You're commenting on old commits